src/main/java/frc/robot/RobotContainer.java

```diff
@@ -43,6 +40,6 @@ private void configureButtonBindings() {}
    */
   public Command getAutonomousCommand() {
     // An ExampleCommand will run in autonomous
-    return m_autoCommand;
+    return null;
```

Comment:
I think we should give the autonomous command here, right? I don't see it being created anywhere else.